src/twist_to_ackermann_node.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 Leidos
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cmath>
+
+#include <ackermann_msgs/msg/ackermann_drive_stamped.hpp>
+#include <geometry_msgs/msg/twist.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+namespace twist_to_ackermann
+{
+auto yaw_rate_to_steering_angle(double speed, double yaw_rate, double wheel_base)
+{
+  if (std::abs(speed) < 0.1 || std::abs(yaw_rate) < 0.1) {
+    return 0.0;
+  }
+
+  return -std::atan(wheel_base / (speed / yaw_rate));
```

Comment:
Where does this (-) come from? Otherwise looks good

Reply:
Added TODO for more investigation.